tests/compile.sh

```diff
@@ -54,7 +54,7 @@ hostname
 
 set +x
 if [[ $MACHINE_ID == macosx.* ]] || [[ $MACHINE_ID == linux.* ]]; then
-  source $PATHTR/modulefiles/${MACHINE_ID}/fv3
+  source $PATHTR/modulefiles/ufs_${MACHINE_ID}
```

Comment:
`MACHINE_ID` also contains compiler, which is a little misleading.
e.g. `MACHINE_ID` = `hera.gnu` or `hera.intel` or `orion.intel` or `wcoss_dell_p3`.
There is a lack of consistency on what `MACHINE_ID` is supposed to mean.
Sometimes it is `machine`.`compiler` and sometimes it is just `machine`.

Reply:
Oh yes, that is "historical", we have moved everything except the wcoss platforms to contain compiler information. I don't know why this didn't happen for wcoss. Agreed that machine_id is not the best name for it, but as long as we are consistent ... we could go through all scripts (and regression test files) and rename it to "platform" maybe? In a separate PR ...

Reply:
I think the changes of making consistent MACHINE_ID might not be trivial, it's better to do it in separate PR.